output/typescript/types.ts

```diff
@@ -17843,11 +17848,16 @@ export interface SecurityActivateUserProfileRequest extends RequestBase {
 
 export type SecurityActivateUserProfileResponse = SecurityUserProfileWithMetadata
 
+export interface SecurityAuthenticateAuthenticateApiKey {
```

Comment:
nit: I know it's generated automatically, but `SecurityAuthenticateAuthenticateApiKey` has `Authenticate` repeated twice. Would be nice if we could avoid this.

Reply:
Ignore me! Did not realise this is in `types.ts`.